src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,40 @@
 import React from 'react';
+import { Post } from '../../Types/Post';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    user,
+    body,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by '}
+          {user && <UserInfo user={user} />}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">{body}</p>
+
+      <hr />
+
+      {post.comments.length === 0 ? (
+        <b data-cy="NoCommentsMessage">No comments yet</b>
+      ) : (
+        <CommentList comments={comments} />
+      )}
```

Comment:
![image](https://user-images.githubusercontent.com/63306833/209977450-98665efa-8ee6-4b91-b3d0-bcea2dc1e0ae.png)

Reply:
fixed it, looks like "ctrl+f" worked incorrect , becouse all was ok in previous commit